src/exchange/services/exchange.state.service.js

```diff
@@ -22,7 +22,7 @@ angular
         }
 
         static isOk (account) {
-            return ExchangeStates.isState(account, "OK");
+            return ExchangeStates.isState(account, "OK") && account.readyToUse;
```

Comment:
This `readyToUse` property comes from your 2API right ? Couldn't you check in the 2API if the account status uis `OK` as well ?

I don't think this service is used only on email accounts, it could also be used on domains and domains don't have the `readyToUse` property

Reply:
There is a check on 2API for that, I just have to remember to use it! :)
It's stored in account.canBeConfigured property.
Fix it in f46c3fd